common/locales/en/pets.json

```diff
@@ -26,6 +26,7 @@
     "inventoryText": "Click an egg to see usable potions highlighted in green and then click one of the highlighted potions to hatch your pet. If no potions are highlighted, click that egg again to deselect it, and instead click a potion first to have the usable eggs highlighted. You can also sell unwanted drops to Alexander the Merchant.",
     "food": "Food and Saddles",
     "noFood": "You don't have any food or saddles.",
+    "dropsExplanation": "Get these items faster with gems if you don't want to wait for them to drop when completing a task. <a href=\"http://habitrpg.wikia.com/wiki/Drops\">Learn more.</a>",
```

Comment:
I think it should say "Learn more about the drop system" to be clear that the link is pointing to the wiki article about the drop system, and not an article about buying things with gems.